api/v1alpha3/condition_const.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+
+// OpenstackCluster Conditions and Reasons.
+const (
+	// NetworkInfrastructureReadyCondition reports of current status of cluster infrastructure.
+	NetworkInfrastructureReadyCondition = "NetworkInfrastructureReady"
+	// LoadBalancerProvisioningReason API Server endpoint for the loadbalancer.
+	LoadBalancerProvisioningReason = "LoadBalancerProvisioning"
+	// LoadBalancerProvisioningFailedReason used for failure during provisioning of loadbalancer.
+	LoadBalancerProvisioningFailedReason = "LoadBalancerProvisioningFailed"
+	// NamespaceNotAllowedByIdentity used to indicate cluster in a namespace not allowed by identity.
+	NamespaceNotAllowedByIdentity = "NamespaceNotAllowedByIdentity"
+)
+
+// OpenStackMachine Conditions and Reasons.
+const (
+	// MachineRunningCondition reports on current status of the Openstack Machine.
+	MachineRunningCondition clusterv1.ConditionType = "MachineRunning"
```

Comment:
Please rename `Machine` to `Instance`.

Reply:
Hi, as per comments on this PR, we had changed Instance to Machine : https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/952

Reply:
Sorry for the late join. The change to `Instance` is totally fine for me. My thought was, that we're also using `Machine` and `OpenStackMachine`. But, to reduce confusion `Instance` is indeed the better choice here.